.github/workflows/pull_request.yml

```diff
@@ -9,6 +9,7 @@ env:
     PYCURL_SSL_LIBRARY: openssl
     ROBOTTELO_BUGZILLA__API_KEY: ${{ secrets.BUGZILLA_KEY }}
     ROBOTTELO_JIRA__API_KEY: ${{ secrets.JIRA_KEY }}
+    ROBOTTELO_ROBOTTELO__SETTINGS__IGNORE_VALIDATION_ERRORS: true
```

Comment:
Should we relly ignore the validation errors in checks? Whats the place time to validate them then ?

Reply:
I wanted to enable the validation of the production configuration. To enable validation in checks, we would need to spend time fine-tuning the `.yaml.template` config files here in the `conf` directory.
This can be a separate effort.

Reply:
Agree, We have to fine-tune them all to be able to turn it on here.

Reply:
Created https://github.com/SatelliteQE/robottelo/issues/15443 for tracking purpose.